client/task-list/style.scss

```diff
@@ -32,7 +32,7 @@
 				height: 32px;
 			}
 
-			.woocommerce-list__item-before svg {
+			.woocommerce-list__item-before .woocommerce-task__icon svg {
```

Comment:
Make this more specific so that it doesn't overwrite the default styling for icons else where in the task list. As I am using a wordpress/icon in the products task.